src/auth0/handlers/default.js

```diff
@@ -122,7 +122,9 @@ export default class DefaultHandler {
 
     // Process Deleted
     if (del.length > 0) {
-      const shouldDelete = this.config('AUTH0_ALLOW_DELETE') === 'true' || this.config('AUTH0_ALLOW_DELETE') === true;
+      const allowDelete = this.config('AUTH0_ALLOW_DELETE') === 'true' || this.config('AUTH0_ALLOW_DELETE') === true;
```

Comment:
this seems like it would be easier to read as one of these. (feel free to disagree)

```
const allowDelete = ['true', true].includes(this.config('AUTH0_ALLOW_DELETE'));
```

OR

```
const allowDelete = String(this.config('AUTH0_ALLOW_DELETE')) === 'true';
```

Reply:
To me that seems like a fancy way to do the same. Wouldn't say it's easier to read/understand. But if you feel that it's better for readability, i'll change it.

Reply:
I retract my suggestions